flags/tests/test_settings.py

```diff
@@ -50,6 +52,11 @@ def test_check_state_no_conditions(self):
         flag = Flag('MY_FLAG', {})
         self.assertFalse(flag.check_state())
 
+    def test_check_state_multiple_conditions(self):
+        request = namedtuple('Request', ['path'])(path='/foo')
```

Comment:
Any reason why you're using a `namedtuple` here and not just a `Mock(path='/foo')`?